tokens/tokens.go

```diff
@@ -0,0 +1,19 @@
+package token
+
+import (
+	"errors"
+	"time"
+)
+
+type Token struct {
+	User     string    `json :"user"`   //define usernya
+	Expired  time.Time `json:"expired"` //define timenya.
+	Id_admin int       `json:"id_admin"`
+}
+
+func (t *Token) Valid() error {
+	if t.Expired.Before(time.Now()) { //kalau sebelum saat ini, token expired
```

Comment:
```suggestion
	if t.Expired.Before(time.Now()) && t.Expired.Equal(time.Now())  { //kalau sebelum saat ini, token expired
```
so that if the token expired exactly right now, it will still be invalid